source/release-notes/v1.4-release-notes.rst

```diff
@@ -0,0 +1,128 @@
+.. _v1.4-release-notes:
+
+v1.4 Release Notes
+==================
+
+Highlights in 1.4:
+
+- `Upgrade to Ruby 2.4, NodeJS 6, Passenger 5`_
+- Security Enhancement: Enable whitelisting directories users can navigate to in files app
+- Security Enhancement: App development mode is disabled by default
+- Security Enhancement: Disable SSH to localhost on the webnode
+- `Fixed copy paste issues in the Shell app`_
+- `Optional Quota warnings on dashboard`_
+- `Experimental SGE/UGE support`_
+- `Add ability to include Javascript in Batch Connect apps`_
+
+
+Upgrading from v1.3
+-------------------
+
+To upgrade run:
+
+TODO is there a way to manage yum repos such that the dependencies are resolved without explicit installs?
+
+.. code-block:: bash
+
+    sudo yum install -y https://yum.osc.edu/ondemand/1.4/ondemand-release-web-1.4-1.el7.noarch.rpm
+    sudo yum clean all
+    sudo yum update ondemand
+
+    # Optionally remove dependencies from prior versions of OOD
+    yum remove rh-nodejs010\* rh-passenger40\* rh-ruby22\*
```

Comment:
Also need to remove `nginx16\*`.

Reply:
@ericfranz @MorganRodgers When we are ready to test 1.4 further at OSC I can run this `yum remove` command on OSC test nodes to ensure what we're advising doesn't cause issues.

Reply:
I don't think rh-ror41 was ever used outside of OSC.

Reply:
This is what I'm running to clean up OSC dev and test systems, minor change from what's in this PR:

```
yum remove nodejs010\* rh-passenger40\* rh-ruby22\* nginx16\*
```

For OSC systems it turns out the above will also remove `rh-ror41`.